File: src/main/java/console/Args.java
package console;

import com.beust.jcommander.Parameter;

import java.util.ArrayList;

public class Args {

    @Parameter(names = "-r", description = "Searching in All subdirectories")
    public boolean r = false;

    @Parameter(names = "-d", description = "Name of directory")
    public String dPath = null;

    @Parameter(description = "File name", required = true)
    public ArrayList name = new ArrayList();

}

File: src/main/kotlin/console/Args.kt
package console

import com.beust.jcommander.Parameter

class Args {
    @Parameter(names = ["-r"], description = "Check subdirectories")
    var checkSubdirectories: Boolean = false

    @Parameter(names = ["-d"], description = "Directory for finding")
    var directory: String? = null

    @Parameter(description = "File name for finding", required = true)
    var filename: List<String> = mutableListOf()
}